epson_patch.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python
+
+#
+# Patch the binary in place: it want to dlopen stuff in /usr
```

Comment:
IANAL, but wouldn't this patch violate [their EULA](https://download.ebz.epson.net/la/linux/inkjet_for_linux.html#softlicense)? This section in specific (emphasis mine):

>3. Other Rights and Limitations.
>You agree **not to modify**, adapt or translate the Software and further agree not to attempt to reverse engineer, decompile, disassemble or otherwise attempt to discover the source code of the Software.

Reply:
I'd say it somewhat acceptable given the modification is done to make the software functional.